### DISCUSSION TITLE
Upgraded, all seemed well..

### DISCUSSION THREAD

**Discussion by RandHoppe** [YOURLS/YOURLS#3810] [Q&A]
...but, the link table is not showing in the admin. New links can be added, but without visual confirmation, (I saw them via phpmyadmin)

kept the old .htaccess in root
and the config.php in user

activated debug, but nothing shows. only on log in.
server logs not offering anything

links still work

Please forgive me, but can anyone offer any suggestions as to where I could look to resolve this? Is there more information I can share that would?

Oh, rats, I just looked at another install, using 1.9.1 and it has the same problem.

Thanks,

- Rand

**Comment by dgw** [comment]
New links work, or only links that existed before upgrading?

The link table in the admin area is not showing _at all_, or is not showing newly-added links?

From which version did you upgrade, to which version, and did you run the updater script yet? (It's not always required, but I must ask.)

There are a few different strange issues that could be happening here, but I'm not yet grasping exactly what behavior you see.
